app/controllers/api/v1x0/orders_controller.rb

```diff
@@ -1,3 +1,4 @@
+require 'api/v1x0/catalog'
```

Comment:
I know we talked about this before - did this end up being required in these controllers?